core/plugins/NtupleWriter.cc

```diff
@@ -826,10 +827,26 @@ bool NtupleWriter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup) {
        iEvent.getByToken(stablegenparticle_token,packed);
 
        for(size_t j=0; j<packed->size();j++){
-
+	 bool skip_particle = false;
 	 const pat::PackedGenParticle* iter = &(*packed)[j];
-	 if(iter->status()!=1) continue;
+	 if(iter->status()!=1) cout<<"iter->status() = "<<iter->status()<<endl;
```

Comment:
Is this a leftover testing cout?

Reply:
Indeed, didn't remove the lines after the testing. Thank you for pointing it out!